### ISSUE TITLE
Is it possible to test on custom data without using Kitti Dataset

### ISSUE THREAD

**Issue by prsnkmr** [maudzung/Complex-YOLOv4-Pytorch#66]
I want to run this model on a single point cloud, without using kitti dataset is it possible?.If it is possible please help me to test this model on point clouds

**Comment by regenthor** [comment]
I tried that too, but failed. Anyone know? @maudzung

**Comment by aorait** [comment]
Yes you can test it using a point cloud stored in a bin file. To do it cleanly you can modify the test.py and dataloader to only use Lidar data. A more hacky solution is to create another folder with the same directory structure as kitti, copy any kitti image and rename it to the same serial id as your custom point clouds id, put this id in the `ImageSets/test.txt`, update the `configs.dataset_dir` in the `test.py` and you should see some detections using this command `python test.py --gpu_idx 0 --pretrained_path your_path/Model_complexer_yolo_epoch_200.pth --show_image`. The results don't seem great though (at least for me since I don't use velodyne data).

**Comment by SAGARCHRY0777** [comment]
can you give the drive link for the pretrained modal
i need the pretrained modal to check the test data
i am not able to see the proper bounding box beacuse i dont have a proper model i believe
my cli is
 python test_1.py --gpu_idx 0 --pretrained_path ../checkpoints/complex_yolov4/complex_yolov4_mse_loss.pth --cfgfile ./config/cfg/complex_yolov4.cfg --show_image --nms_thresh 0.2 --conf_thresh 0.2
i have this .pth file and i am keeping the threshold even too low
i am still not able to get the proper bounding box
